### PR TITLE
modified bfs,dfs

### PR DESCRIPTION
**Pull request by Jugalcody**

### CODE REVIEW

**Comment by Jugalcody** [comment]
have to add a statement in the automated test case as updated to remove the error

File: easyPythonpi/methods/search.py [hunk 1]
```diff
@@ -1,37 +1,46 @@
 #!/usr/bin/env python
 #-*- coding: utf-8 -*-
-
+import sys,os
+sys.path.append(f'{os.getcwd()[:-4]}')
```
Comment (extinctsion) [inline easyPythonpi/methods/search.py:4]: pls dont use hardcoded path as it will never work in cloud

File: easyPythonpi/methods/search.py [hunk 2]
```diff
@@ -1,37 +1,46 @@
...
-            result.append(node)
-            queue.extend(set(graph.get(node, [])) - visited)
-
-    return result
+visited = []
+queue = []   
+resultbfs=[]
+resultdfs=[]
+visitedbfs=[]
+visiteddfs=[]
```
Comment (extinctsion) [inline easyPythonpi/methods/search.py:14]: why so many extra spaces? very inefficient.

**Comment by extinctsion** [comment]
There are failed tests. Hence, we cannot merge this PR. Thanks for contributing in the open source community.

**Pull request closed without merging**